simulationdataschema/model_system_review.py

```diff
@@ -0,0 +1,865 @@
+
+    Returns:
+        (AtomicCell): The AtomicCell section resolved from the parent.
+    """
+    if section.m_parent is None and section.m_parent.atomic_cell is None:
+        logger.error(
+            "Could not find m_parent ModelSystem and its AtomicCell section for "
+            "Symmetry analyzer."
+        )
+        return
+    return section.m_parent.atomic_cell[0]
+
+
+class AtomicCell(GeometricSpace):
+    """
+    A base section used to specify the atomic cell quantities (labels, positions) of a system
+    at a given moment in time.
+    """
+
+    name = Quantity(
+        type=MEnum("original", "primitive", "standard"),
+        description="""
+        Name to identify the cell structure. It might be:
+            - 'original' as in orignally parsed,
+            - 'primitive' as the primitive unit cell,
+            - 'standard' as the standarized cell used for referencing.
+        """,
+    )
+
+    labels = Quantity(
+        type=str,
+        shape=["*"],
+        description="""
+        List containing the labels of the atomic species in the system at the different positions
+        of the structure. It refers to a chemical element as defined in the periodic table,
+        e.g., 'H', 'O', 'Pt'. This quantity is equivalent to `atomic_numbers`.
+        """,
+    )
+
+    atomic_numbers = Quantity(
+        type=np.int32,
+        shape=["*"],
+        description="""
+        List of atomic numbers Z. This quantity is equivalent to `labels`.
+        """,
+    )
+
+    positions = Quantity(
+        type=np.float64,
+        shape=["*", 3],
+        unit="meter",
+        description="""
+        Positions of all the atoms in Cartesian coordinates.
+        """,
+    )
+
+    lattice_vectors = Quantity(
+        type=np.float64,
+        shape=[3, 3],
+        unit="meter",
+        description="""
+        Lattice vectors of the simulated cell in Cartesian coordinates. The first index runs
+        over each lattice vector. The second index runs over the $x, y, z$ Cartesian coordinates.
+        """,
+    )
+
+    lattice_vectors_reciprocal = Quantity(
+        type=np.float64,
+        shape=[3, 3],
+        unit="1/meter",
+        description="""
+        Reciprocal lattice vectors of the simulated cell, in Cartesian coordinates and
+        including the $2 pi$ pre-factor. The first index runs over each lattice vector. The
+        second index runs over the $x, y, z$ Cartesian coordinates.
+        """,
+    )
+
+    periodic_boundary_conditions = Quantity(
+        type=bool,
+        shape=[3],
+        description="""
+        If periodic boundary conditions are applied to each direction of the crystal axes.
+        """,
+    )
+
+    velocities = Quantity(
+        type=np.float64,
+        shape=["*", 3],
+        unit="meter / second",
+        description="""
+        Velocities of the atoms. It is the change in cartesian coordinates of the atom position
+        with time.
+        """,
+    )
```

Comment:
Yeah I agree, I have always felt this way. I understand why someone would place forces under output but in practice I think it makes more sense with positions. Output is more for collective properties. Are there any other properties besides forces and acceleration that are stored as a function on individual atoms in calculation?

For the MD parsers it is actually sort of a pain because forces are always kept with positions and velocities in the same file, while the other output is stored elsewhere or parsed separately.

Reply:
The former. It's because the `metainfo` / `data` will follow the order in which we declare the schema quantities, so I would group common kinds together.

Reply:
> Are there any other properties besides forces and acceleration that are stored as a function on individual atoms in calculation?

I can only think of derived tensors, e.g. stress and strain.

Reply:
Ok, I moved `velocities` after `positions`, but before defining `forces`, let's think a bit more about `AtomType` or `Atoms`. I wrote a TODO